### PR TITLE
Upgrade to React Native 0.49.3 (take two)

### PR DESCRIPTION
**Pull request by danieljharvey**
Recreated iOS and Android projects this time to try and avoid test build issues.

### CODE REVIEW

**Comment by tino-junge** [comment]
Looks good already while reading through the changes 👌 Thanks for upgrading. Will check it out in more detail the next days and get it merged as soon as possible.

File: package.json [hunk 1]
```diff
@@ -40,12 +42,12 @@
...
     "lodash": "^4.11.0",
     "moment": "^2.12.0",
-    "react": "~15.4.2",
-    "react-dom": "~15.4.2",
-    "react-native": "0.42.0",
-    "react-native-vector-icons": "^4.0.0",
-    "react-navigation": "^1.0.0-beta.9",
-    "react-redux": "^4.4.5",
+    "prop-types": "^15.6.0",
+    "react": "^16.0.0-beta.5",
```
Comment (tino-junge) [inline package.json]: why not the actual stable react 16 release ? https://github.com/facebook/react/releases

Comment (tino-junge) [reply package.json]: Looks like peer dependency ist still the react 16 beta. Then let's upgrade to react 16 stable as soon as it's support e.g. with RN 50

**Comment by tino-junge** [comment]
@danieljharvey before merging, could you please update the React Native Version in the Readme file as well?

File: package.json (hunk 1 above)
Comment (danieljharvey) [reply package.json]: Aye - I was a bit surprised they paired a beta React version with a release RN - but yes, let's update as soon as possible.

**Pull request merged**